kore/src/Kore/JsonRpc.hs

```diff
@@ -88,6 +92,8 @@ data ExecuteRequest = ExecuteRequest
     , maxDepth :: !(Maybe Depth)
     , cutPointRules :: !(Maybe [Text])
     , terminalRules :: !(Maybe [Text])
+    , movingAverageStepTimeout :: !(Maybe Bool)
+    , stepTimeout :: !(Maybe StepTimeout)
```

Comment:
We should document changes/additions [in our RPC documentation](https://github.com/runtimeverification/haskell-backend/blob/master/docs/2022-07-18-JSON-RPC-Server-API.md) so we don't lose track of them.